.github/workflows/ci.yml

```diff
@@ -23,6 +23,11 @@ jobs:
 
       - name: eslint
         run: npm run lint
+      
```

Comment:
You might have to change the checkout step above this to get the right commit for CodeQL, based on what their example was doing (fetch-depth 2, and if it's a PR then checking out HEAD^2)